telemetry/jetbrains/gradlew

```diff
@@ -159,14 +175,9 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-APP_ARGS=$(save "$@")
+APP_ARGS=`save "$@"`
```

Comment:
This is a stylistic regression, backticks are harder to read, can't be nested, and not part of POSIX, so not a "best practice"

Reply:
This is gradle editing gradle's file on update, this file is not manually touched